contracts/ETHRegistrarController.sol

```diff
@@ -23,30 +27,38 @@ contract ETHRegistrarController is Ownable {
         keccak256("register(string,address,uint256,bytes32)") ^
         keccak256("renew(string,uint256)")
     );
-
     bytes4 constant private COMMITMENT_WITH_CONFIG_CONTROLLER_ID = bytes4(
         keccak256("registerWithConfig(string,address,uint256,bytes32,address,address)") ^
         keccak256("makeCommitmentWithConfig(string,address,bytes32,address,address)")
     );
+    bytes4 constant public BULK_RENEWAL_ID = bytes4(
+        keccak256("rentPrice(string[],uint)") ^
+        keccak256("renewAll(string[],uint")
+    );
 
-    BaseRegistrar base;
-    PriceOracle prices;
+    // Base Registrar contract
+    BaseRegistrar public base;
+    // Price oracle contract
+    PriceOracle public prices;
+    // Minimum and maximum commitment ages, in seconds.
     uint public minCommitmentAge;
     uint public maxCommitmentAge;
-
+    // Referral fee, in 1/1000ths
```

Comment:
So if it's set as 1, it means 0.1 %, 100 for 10%, etc?

Reply:
That's right.